lib/api/tracking_manager.dart

```diff
@@ -5,6 +5,8 @@ import 'package:flagship/flagship.dart';
 import 'package:flagship/hits/hit.dart';
```

Comment:
what is the diff between file lib/api/trackingManager.dart  and lib/api/tracking_manager.dart

Reply:
oh no, an old file should't be here , still exist after the renaming files i guess . i remove the **trackingManager** one